Makefile

```diff
@@ -0,0 +1,23 @@
+GIT_REF=$(shell sh -c "git rev-parse --short HEAD")
+TOWER_OPENSHIFT_PROJECT=tower-e2e-${GIT_REF}
+
+.PHONY test/e2e:
+test/e2e:
+	echo "Installing tower on openshift: ${OPENSHIFT_MASTER}"
+	ansible-playbook -i ./inventories/hosts \
+	playbooks/install_tower.yml \
+	-e tower_openshift_master_url=${OPENSHIFT_MASTER} \
+	-e tower_openshift_username=${TOWER_OPENSHIFT_USERNAME} \
+	-e tower_openshift_password=${TOWER_OPENSHIFT_PASSWORD} \
+	-e tower_openshift_project=${TOWER_OPENSHIFT_PROJECT} \
+	-e tower_openshift_pg_pvc_size=10Gi \
+	--extra-vars e2e_test_config.yml
+	echo "Bootstrapping tower in openshift project: ${TOWER_OPENSHIFT_PROJECT}"
+	ansible-playbook -i ./inventories/hosts \
+	playbooks/bootstrap.yml \
+	-e e2e_tower_license=${TOWER_LICENSE} \
+	-e e2e_tower_username=${TOWER_USERNAME} \
+	-e e2e_tower_password=${TOWER_PASSWORD} \
+	-e tower_openshift_project=${TOWER_OPENSHIFT_PROJECT} \
+	--extra-vars e2e_test_config.yml
+	oc delete project ${TOWER_OPENSHIFT_PROJECT}
```

Comment:
You might want to consider converting this to a bash script that gets called from the make target (test/e2e.sh). You could use the trap command to implement a crude, possibly optional, cleanup function to enforce `oc delete project ${TOWER_OPENSHIFT_PROJECT}`.

See here for an example https://github.com/integr8ly/installation/blob/master/scripts/release.sh#L88

This will also be easier to expand in the future if the need should arise.

Reply:
Sounds good. Cleanup was on my hit list but wasn't sure how best to do it. I'll take this approach. :)